<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Optimize</class>
 <widget class="QDialog" name="Optimize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AvisionR - Optimize</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(255, 255, 255);
background-color: #262626;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>61</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Area Code:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Area">
       <property name="minimumSize">
        <size>
         <width>251</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>251</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #686868;</string>
       </property>
       <property name="maxLength">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>City:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_City">
       <property name="minimumSize">
        <size>
         <width>281</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #686868;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Zip Code:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_ZipCode">
       <property name="minimumSize">
        <size>
         <width>261</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #686868;</string>
       </property>
       <property name="maxLength">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>State:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_State">
       <property name="minimumSize">
        <size>
         <width>271</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #686868;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QCommandLinkButton" name="commandLinkButton">
     <property name="text">
      <string>Optimize</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_Area</tabstop>
  <tabstop>lineEdit_City</tabstop>
  <tabstop>lineEdit_ZipCode</tabstop>
  <tabstop>lineEdit_State</tabstop>
  <tabstop>commandLinkButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
